docs/CLAVIN.md

```diff
@@ -0,0 +1,51 @@
+# Description
+
+[CLAVIN (Cartographic Location And Vicinity INdexer)](https://github.com/Berico-Technologies/CLAVIN) is an
+award-winning open source software package for document geotagging and geoparsing that employs context-based
+geographic entity resolution.
+
+It extracts location names from unstructured text and resolves them against a gazetteer to produce data-rich
+geographic entities.
+
+CLAVIN does not simply "look up" location names – it uses intelligent heuristics to identify exactly which
+"Springfield" (for example) was intended by the author, based on the context of the document. CLAVIN also employs
+fuzzy search to handle incorrectly-spelled location names, and it recognizes alternative names (e.g., "Ivory Coast"
+and "Côte d'Ivoire") as referring to the same geographic entity.
+
+By enriching text documents with structured geo data, CLAVIN enables hierarchical geospatial search and advanced
+geospatial analytics on unstructured data.
+
+# Prerequisites
+
+The Lumify CLAVIN plugin will resolve already identified [term mentions](../../../docs/glossary.md#term-mention).
+To get term mentions you will need to install a graph property worker plugin that identifies term mentions, such as
+the opennlp-me-extractor plugin.
+
+# Build
+
+The following instructions can be used to build the Lucene index of geonames data that CLAVIN uses.
+
+## Build
+[Follow the CLAVIN site to build the CLAVIN:] 
+(https://github.com/Berico-Technologies/CLAVIN#how-to-build--use-clavin)
+
+## Install Locally
+
+    mkdir -p /opt/lumify/clavin-index
+    mv CLAVIN/IndexDirectory/* /opt/lumify/clavin-index
+
```

Comment:
This can be simplified to `mv CLAVIN/IndexDirectory /opt/lumify/clavin-index`. We might also want to put in a step to remove the old index: `rm -rf /opt/lumify/clavin-index` prior to moving the new one in place.